packages/search-ui-engines-connector/src/handlers/search/Request.ts

```diff
@@ -0,0 +1,58 @@
+import type { Filter, QueryConfig, RequestState } from "@elastic/search-ui";
+import { MixedFilter } from "@searchkit/sdk";
+import { helpers } from "@elastic/search-ui";
+import { isValidDateString } from "./Configuration";
+
+export interface SearchkitVariables {
+  query: string;
+  filters: MixedFilter[];
+  from: number;
+  size: number;
+  sort: string;
+}
+
+export function getFilters(
+  filters: Filter[] = [],
+  baseFilters: Filter[] = []
+): MixedFilter[] {
+  return filters.reduce((acc, f) => {
+    const isBaseFilter = baseFilters.find((bf) => bf === f);
```

Comment:
nit: i think this could be `baseFilters.includes(f)`

Reply:
agreed. Thanks!